hw/acpi/aml-build.c

```diff
@@ -41,7 +41,10 @@
 #define PCI_HOST_BRIDGE_IO_0_MIN_ADDR      0x0000
 #define PCI_HOST_BRIDGE_IO_0_MAX_ADDR      0x0cf7
 #define PCI_HOST_BRIDGE_IO_1_MIN_ADDR      0x0d00
-#define PCI_HOST_BRIDGE_IO_1_MAX_ADDR      0xffff
+#define PCI_HOST_BRIDGE_IO_1_MAX_ADDR      0x9fff
```

Comment:
@liujing2 longer term we should decide if these constants should even be in this file. Ideally they should be driven from the platform definition (in our case virt). So piix and q35 and virt should define these values in a platform specific file.

Reply:
@mcastelino If only virt uses multi-segment PCI, agree with you. piix and q35 own the whole MIN~MAX ADDR and virt separates them for each segment.